sml/src/sml_list.c

```diff
@@ -112,16 +112,64 @@ sml_list *sml_list_init() {
 
 void sml_list_add(sml_list *list, sml_list *new_entry) { list->next = new_entry; }
 
+/* Note: this gets passed _every_ object found */
```

Comment:
"invoked for _every_ sml object encountered"?

("passed every object" can't be right because it's only passed a single object in each invocation,
 "found" can't be right because we don't search, we process/decode input...)

Reply:
yeah I guess